FeedbackSample/FeedbackSample/foresee_configuration.json

```diff
@@ -16,5 +16,11 @@
       }
     ],
     "surveyCompletionPageDelay":3
+  },
+  "notificationType": "IN_SESSION",
+  "measures": {},
+  "invite":
+  {
+    "baseColor": [237, 38, 54]
```

Comment:
I'm not sure if this is the right fix for this tbh... If the client choose to only use the feedback feature, why do they have to include a cxMeasure invite setup? I feel we should update the sdk's warning message copies instead of updating the sample app. What do you think @wayneburkett ?

Reply:
👍